package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pimp-my-sql",
-  "version": "0.1.3",
+  "version": "0.2.0",
```

Comment:
Because I feel the .1.x branch is toast due to the fact this bug fixes search queries that do not have JOIN clauses.

Currently any search query with no JOIN clause will squash all existing where clauses.

This will change output (fixing it), but this is a larger change than a patch change as the output of identical code running this library will change, but not an API breaking change so no major version jump.